### PR TITLE
up lift to v3-m1

### PR DESCRIPTION
**Pull request by riveramj**
version bump to allow for compilation

### CODE REVIEW

**Comment by arigoldx** [comment]
say, I thought that instead of changing (bumping?) the lift version, that we needed to up the scala version?

also dunno why i laughed when i saw this non-elemican PR but, heck, i'll take it. more laughter = better.

**Comment by riveramj** [comment]
the at lift snapshot isnt published anymore. The most recent release is Milestone 6, which requires the scala 2.11. Bumping to 2.11 tho requires a newer specs version, since the old one isnt supported on 2.11.

**Comment by arigoldx** [comment]
:+1: it mosdef works now!

File: build.sbt [hunk 1]
```diff
@@ -16,12 +16,12 @@ seq(webSettings :_*)
 scalacOptions ++= Seq("-deprecation", "-unchecked", "-feature", "-Xfatal-warnings")
 
 libraryDependencies ++= {
-  val liftVersion = "3.0-SNAPSHOT"
+  val liftVersion = "3.0-M6"
```
Comment (Shadowfiend) [inline build.sbt]: I'd leave this on `SNAPSHOT` for now. There's no real stability goal in `lift-jungle-gym` atm, and Lift 3 should go final in the next couple of months with any luck.

Comment (riveramj) [inline build.sbt]: `3.0-SNAPSHOT` isnt published currently. Tho there seems to be a M7.

screen shot 2016-01-24 at 6 19 26 pm

Comment (riveramj) [inline build.sbt]: nvm ignore me

Comment (riveramj) [inline build.sbt]: @Shadowfiend 
i can resolve this via browser but sbt isnt having any of it.
```
[warn] ==== snapshots: tried
[warn]   http://oss.sonatype.org/content/repositories/snapshots/net/liftweb/lift-webkit_2.11/3.0-SNAPSHOT/lift-webkit_2.11-3.0-SNAPSHOT.pom
```

**Comment by Shadowfiend** [comment]
@riveramj Your resolution issue seems to be local—I'd say see if you can blow away your local ivy repo and maybe your ivy cache for Lift. I just resolved it properly, so I'm going to go ahead and send this on in :)

**Comment by arigoldx** [comment]
I know this is closed but thought I'd mention that if I leave `liftVersion` as `3.0-SNAPSHOT` then `sbt` can't start. If I change it to `3.0-M6` then it's A OK.

I even blew out my `~/.ivy2`, which I'm assuming is what is necessary to

> blow away your local ivy repo and maybe your ivy cache for Lift..

but might not be.

For completeness, here's the error:

```
sbt.ResolveException: unresolved dependency: net.liftweb#lift-webkit_2.11;3.0-SNAPSHOT: not found
```

**Comment by Shadowfiend** [comment]
I bet we may be missing the sonatype snapshots resolver in the build file.

**Comment by arigoldx** [comment]
? ain't that what this is?

```
resolvers ++= Seq(
  "snapshots" at "http://oss.sonatype.org/content/repositories/snapshots",
  "releases"  at "http://oss.sonatype.org/content/repositories/releases"
)
```

**Comment by Shadowfiend** [comment]
Hmmm… Sure is. Curiouser and curiouser…

**Comment by Shadowfiend** [comment]
Do you have this issue with other Lift 3-based projects like https://github.com/lift/lift_30_sbt ?

**Comment by riveramj** [comment]
I think this is the same issue I had before, which is why i left it as M#
and not snapshot. Didnt dig any further, but can confirm this isnt just
ari's machine.

**Pull request merged**